File: src/main/java/com/thetaris/goldprice/GoldpriceApplication.java
package com.thetaris.goldprice;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;

@SpringBootApplication
@EnableConfigurationProperties
public class GoldpriceApplication {

	public static void main(String[] args) {
		SpringApplication.run(GoldpriceApplication.class, args);
	}

}

File: src/main/kotlin/com/thetaris/goldprice/GoldPriceApplication.kt
package com.thetaris.goldprice;

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableConfigurationProperties
class GoldPriceApplication

fun main(args: Array<String>) {
    runApplication<GoldPriceApplication>(*args)
}
